pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -629,10 +629,9 @@ func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, devi
 			resizeStatus, resizeErr := resizer.Resize(devicePath)
 
 			if resizeErr != nil {
-				resizeDetailedError := volumeToMount.GenerateErrorDetailed("MountVolume.resizeFileSystem failed", resizeErr)
+				resizeSimpleError, resizeDetailedError := volumeToMount.GenerateError("MountVolume.resizeFileSystem failed", resizeErr)
 				glog.Error(resizeDetailedError)
-				og.recorder.Eventf(volumeToMount.Pod, v1.EventTypeWarning, kevents.FileSystemResizeFailed, resizeDetailedError.Error())
```

Comment:
We needn't to record this error as the outside code will record it.

Reply:
Which outside code will record the event?

Reply:
The `eventRecorderFunc` in the outside code will record an event if there is any error during the `mountVolumeFunc`, included the `resizeFileSystem` error, see [this line](https://github.com/kubernetes/kubernetes/blob/8e4084e4d5c7b79c6c2f7e814be02f78ca29b999/pkg/volume/util/operationexecutor/operation_generator.go#L576). So we needn't to record any event inside `resizeFileSystem` if we throw the error to the outside code.

Reply:
oh good catch. I missed that line. thanks